libs/live-config/src/index.ts

```diff
@@ -0,0 +1 @@
+export * from "./FeatureFlags";
```

Comment:
The folder should follow _snakeCase_ naming convention

```suggestion
export * from "./featureFlags";
```

Reply:
fixed